extension/lib/doofinder/Client.js

```diff
@@ -4,19 +4,22 @@ const { promisify } = require('util')
 class Client {
   /**
    * @param {Object} config
-   * @param {string} authHeader
    */
   constructor ({config, tracedRequest, log}) {
     this.baseUri = `https://${config.zone}-search.doofinder.com/5/`
```

Comment:
The url is `https://eu1-search.doofinder.com/5/search`. The `/search` sufix is missing

Reply:
thats just the base uri. we are using 2 endpoints, therefore we cannot set it here